quotes/tests/tests.py

```diff
@@ -326,9 +312,9 @@ def setUp(self):
         self.user.save()
         self.quote = Quote(
             text="Some quote text", reported_by=self.user, id=124
-        )
+        ) #Does self.quote need to be a non-factory based quote?^^
```

Comment:
In other words, use a factory instead. If we use a `id` directly like this it could break the tests on random occations if a Quote with that id already exists.